cc/mcts_player.cc

```diff
@@ -123,9 +123,7 @@ Coord MctsPlayer::SuggestMove(int new_readouts, bool inject_noise) {
   // time SuggestMove has been called for a game, or PlayMove was called without
   // a prior call to SuggestMove.
   if (!root_->HasFlag(MctsNode::Flag::kExpanded)) {
-    tree_search_leaves_.clear();
-    SelectLeaves(1, &tree_search_leaves_);
-    ProcessLeaves(tree_search_leaves_);
+    TreeSearch(1);
```

Comment:
Does this not work with options_.virtual_losses?  It should... I'm only vaguely concerned about fixed-batch size models, although i suppose the model impl can take care of padding?